servers/physics_3d/joints/generic_6dof_joint_3d_sw.h

```diff
@@ -134,7 +119,7 @@ class G6DOFTranslationalLimitMotor3DSW {
 	Vector3 m_damping; //!< Damping for linear limit
 	Vector3 m_restitution; //! Bounce parameter for linear limit
 	//!@}
-	bool enable_limit[3];
+	bool enable_limit[3] = { true, true, true };
 
 	G6DOFTranslationalLimitMotor3DSW() {
```

Comment:
All the inits here can also be moved to the struct definition. Calculations are trivial so you can apply them to get constants (`Vector3(1, 1, 1) * 0.7f` -> `Vector3(0.7, 0.7, 0.7)`).